cmd/wallet-web/src/components/Vaults/DeleteVaultModal.vue

```diff
@@ -0,0 +1,115 @@
+<!--
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
+<template>
+  <modal :show="showModal">
+    <template #content>
+      <!--content-->
+      <div class="flex relative flex-col items-center px-8 pt-10 w-full">
+        <div class="flex justify-center items-center w-15 h-15 bg-primary-valencia rounded-full">
+          <svg width="32" height="32" xmlns="http://www.w3.org/2000/svg">
+            <g transform="translate(3 6)" fill="none" fill-rule="evenodd">
+              <rect stroke="#ffffff" stroke-width="2" x="1" y="1" width="24" height="19" rx="4" />
+              <ellipse fill="#ffffff" cx="8" cy="13" rx="4" ry="2" />
+              <circle fill="#ffffff" cx="8" cy="8" r="2" />
+              <path
+                stroke="#ffffff"
+                stroke-width="2"
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                d="M16 8h5M16 13h2"
+              />
+            </g>
+          </svg>
+        </div>
+        <span class="pt-5 pb-3 text-lg font-bold text-neutrals-dark">
+          {{ t('Vaults.deleteVault') }}?
+        </span>
+        <div class="relative flex-auto">
+          <p class="pb-12 text-base text-center text-neutrals-medium">
+            {{ t('Vaults.deleteVaultConfirmMessage') }}
+          </p>
+        </div>
+      </div>
+    </template>
+    <!-- Buttons Container -->
```

Comment:
Maybe we just keep these comments in the Modal itself and remove from here, the template name is self-explanatory